src/rendering/screenspacerenderable.cpp

```diff
@@ -341,7 +359,18 @@ ScreenSpaceRenderable::ScreenSpaceRenderable(const ghoul::Dictionary& dictionary
         glm::vec4(0.f),
         glm::vec4(1.f)
     )
-    , _delete(DeleteInfo)
+    , _delete(DeleteInfo),
+    , _renderOrder(RenderOrderInfo, 0.f, -1000.f, 1000.f)
+    , _alignment(AlignmentInfo, {
```

Comment:
There is no constructor for the `OptionProperty` that takes in all the options. These will all have to be added later with the `addOption` function.